orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/model/GatewayConfig.java

```diff
@@ -181,32 +188,34 @@ public String getUserFacingKey() {
      */
     public GatewayConfig getGatewayConfig(ServiceEndpointFinder serviceEndpointFinder)
             throws ServiceEndpointLookupException, InterruptedException {
-        return new GatewayConfig(this, getServiceEndpoint(serviceEndpointFinder));
+        return new GatewayConfig(this, getServiceEndpoint(serviceEndpointFinder, !Boolean.TRUE.equals(getUseCcm())));
```

Comment:
It would be quite nice to add a unit test specifically showing how this Boolean logic is expected to work. Despite it being explained in the commit message, I still needed to think about it.

Reply:
Will do in a follow-on.